### PR TITLE
chore: improve providence docs

### PR DESCRIPTION
**Pull request by tlouisse**
## What I did

1.

### CODE REVIEW

File: .changeset/unlucky-rabbits-appear.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@lion/ui': patch
```
Comment (gerjanvangeest) [inline .changeset/unlucky-rabbits-appear.md:2]: Shouldn't this be providence?

Comment (tlouisse) [reply .changeset/unlucky-rabbits-appear.md]: Thanks. Fixed

**Review by gerjanvangeest — APPROVED** [review]

**Review by okadurin — APPROVED** [review]

File: docs/fundamentals/node-tools/providence-analytics/overview.md [hunk 2]
```diff
@@ -23,170 +20,76 @@ For instance for a repo "lion-based-ui" that extends @lion/\* we can answer ques
...
+npx providence analyze match-imports -t /importing/project -r /exporting/project
 ```
 
-### Updating submodules
+## Utils
 
-Please run:
+Providence comes with many tools for deep traversal of identifiers,
+the (babel like) traversal of ast trees in oxc and swc and more.
+Also more generic utils for caching and performant globing come delivered with Providence.
```
Comment (tlouisse) [inline docs/fundamentals/node-tools/providence-analytics/overview.md:85]:
```suggestion
Also more generic utils for caching and performant globbing come delivered with Providence.
```

File: docs/fundamentals/node-tools/providence-analytics/overview.md [hunk 3]
```diff
@@ -23,170 +20,76 @@ For instance for a repo "lion-based-ui" that extends @lion/\* we can answer ques
...
-Run:
+Let's say we run `find-exports`:
 
 ```bash
-providence analyze
+npx providence analyze find-imports
 ```
 
-Now you will get a list of all predefined analyzers:
+Now it retrieves all relevant data about es module exports.
```
Comment (tlouisse) [inline docs/fundamentals/node-tools/providence-analytics/overview.md:48]:
```suggestion
Now it retrieves all relevant data about es module imports.
```

File: docs/fundamentals/node-tools/providence-analytics/overview.md [hunk 4]
```diff
@@ -23,170 +20,76 @@ For instance for a repo "lion-based-ui" that extends @lion/\* we can answer ques
...
-
-Analyzers can be described as predefined queries that use AST traversal.
+- find-imports
+- find-exports
+- find-classes
+- match-imports
+- match-subclasses
 
-Run:
+Let's say we run `find-exports`:
```
Comment (tlouisse) [inline docs/fundamentals/node-tools/providence-analytics/overview.md:42]:
```suggestion
Let's say we run `find-imports`:
```

File: docs/fundamentals/node-tools/providence-analytics/overview.md [hunk 5]
```diff
@@ -23,170 +20,76 @@ For instance for a repo "lion-based-ui" that extends @lion/\* we can answer ques
 
 - etc...
 
-All the above results can be shown in a dashboard (see below), which allows to sort exports from reference project (@lion) based on popularity, category, consumer etc.
-The dashboard allows to aggregate data from many target projects as well and will show you on a detailed (file) level how those components are being consumed by which projects.
+Providence uses AST analysis for getting the most advanced analysis possible.
```
Comment (tlouisse) [inline docs/fundamentals/node-tools/providence-analytics/overview.md:23]:
```suggestion
Providence uses abstract syntax trees (ASTs) to have the most advanced analysis possible.
```

**Pull request merged**